server/embedded/src/org/labkey/embedded/LabKeyServer.java

```diff
@@ -166,11 +167,14 @@ protected TomcatWebServer getTomcatWebServer(Tomcat tomcat)
                     getDataSourceResources(contextProperties, context).forEach(contextResource -> context.getNamingResources().addResource(contextResource));
 
                     // Add extra resources to context (e.g. LDAP, JMS)
-                    getExtraContextResources(contextProperties).forEach(contextResource -> context.getNamingResources().addResource(contextResource));
+                    addExtraContextResources(contextProperties, context);
```

Comment:
Seems like the above getDataSourceResources call and method could use a similar treatment

Reply:
Updated (I debated before, but then thought it should be deleted before too long, once existing configs are migrated)